packages/payload/src/queues/config/types/index.ts

```diff
@@ -118,6 +136,19 @@ export type JobsConfig = {
    * @default false
    */
   runHooks?: boolean
+  /**
+   * Determines how task / workflow schedules should be handled. This
+   * property needs to be set in order for the `schedule` property to work.
+   *
+   * If set to `cron`, the job system will use cron jobs to run scheduled tasks.
+   * If set to `manual`, you are responsible for handling schedules yourself, for example by calling the `/api/payload-jobs/handle-schedules` endpoints
+   *
+   * @remark On serverless platforms like Vercel, you should use `manual` to avoid issues with cron jobs not running as expected.
+   * You can then use Vercel Cron to call the `/api/payload-jobs/handle-schedules` endpoint at specified intervals
+   *
+   * @default 'manual'
+   */
+  scheduler?: 'cron' | 'manual'
```

Comment:
What does the 'manual' option do in the code? If it is cron we run the cron schedule, but otherwise the endpoints are just available?

I'm trying to move quick on reviewing this so maybe I'm missing something.

Reply:
> If it is cron we run the cron schedule, but otherwise the endpoints are just available?

Correct - well, the endpoints are always available. If it is set to `cron`, we create cronjobs on init to handle scheduling. If it is manual, we do not.

An error is thrown if this is explicitly set and schedules are used

Reply:
So this is effectively the same?

`scheduler: 'manual'`
`// scheduler: 'manual'`

Reply:
correct - had it required explicitly like this to ensure the user thinks about what scheduler mode they want to use.